### PR TITLE
Docs for @-mentioning directories

### PR DESCRIPTION
**Pull request by MaedahBatool**
This PR adds docs for @mentioning directories in Cody chat for Enterprise users.

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@chenkc805 can I please get your review?

File: docs/cody/clients/enable-cody-enterprise.mdx [hunk 1]
```diff
@@ -67,6 +67,20 @@ Cody Enterprise supports searching up to 10 repositories to find relevant contex
...
+
+To better support teams working with large monorepos, Enterprise users can `@-mention` directories when chatting with Cody. This helps you define more specific directories and sub-directories within that monorepo to give more precise context.
+
+To do this, type `@` in the chat, and then select **Directories** to search other repositories for context in your codebase.
+
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/%40-mention-dir-0824.mp4" type="video/mp4"/>
+</video>
+
+Also, note that you can only `@-mention` remote directories, not the local directory.
```
Comment (camdencheek) [inline docs/cody/clients/enable-cody-enterprise.mdx:82]:
```suggestion
Also, note that you can only `@-mention` remote directories, not local directories.
```

Comment (camdencheek) [reply docs/cody/clients/enable-cody-enterprise.mdx:82]: Will a user know what a "remote directory" is, or should we say something more like "directories on your Sourcegraph instance"?

**Review by camdencheek — APPROVED** [review]

File: docs/cody/clients/enable-cody-enterprise.mdx (hunk 1 above)
Comment (chenkc805) [reply docs/cody/clients/enable-cody-enterprise.mdx:82]: I think we should say "remote directories", but we can further clarify that this means "directories in your Sourcegraph instance". So I'd say something like: 

> Please note that currently you can only `@-mention` remote directories (directories in your Sourcegraph instance) but not local directories. This means that recent changes to your directories can't be utilized as context until your Sourcegraph instance re-indexes any changes.

> If you want to include recent changes that haven't been indexed in your Sourcegraph instance, you can `@-mention` specific files, lines of code, or symbols.

**Comment by MaedahBatool** [comment]
Thank you folks for the review. Merging it with the docs release branch to go live tomorrow. :)

**Pull request merged**